tron/config/manager.py

```diff
@@ -158,6 +158,9 @@ def __contains__(self, name):
     def get_namespaces(self):
         return self.manifest.get_file_mapping().keys()
 
+    def check_config(self, name, content):
```

Comment:
You can take out the call to `check_config` from the API controller, since this doesn't do anything.

Reply:
Yeah, this can work too. I was assuming that we can add some checks here. But since we will get rid of this soon, removing this call is better.